src/lib/utils/helpers.ts

```diff
@@ -123,3 +124,21 @@ export function sumDiggingFees(archaeologists: Archaeologist[]): BigNumber {
     ethers.constants.Zero
   );
 }
+
+/**
+ * Builds a resurrection date string from a BigNumber
+ * Ex: 09.22.2022 7:30pm (12 Days)
+ * @param resurrectionTime
+ * @returns The resurrection string
+ */
+export function buildResurrectionDateString(
+  resurrectionTime: BigNumber,
+  format = 'MM.DD.YYYY h:mmA'
+): string {
+  const resurrectionDateString = moment.unix(resurrectionTime.toNumber()).format(format);
+  const msUntilResurrection = resurrectionTime.toNumber() * 1000 - Date.now();
+  const timeUntilResurrection = `${msUntilResurrection < 0 && '-'}${moment
```

Comment:
let's swap this out for a ternary because if msUntilResurrection is positive we short circuit and print out "false"

Reply:
Not sure how I overlooked this. Thanks.